srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditMultiSelectVariable.java

```diff
@@ -0,0 +1,168 @@
+     * VariableGroups) to the user
+     */
+    public EditMultiSelectVariable(final ModelInspectionService modelService)
+    {
+        super(modelService);
+        fDisplayType = MultiSelectVariable.DisplayType.Radio;
+        // Start with one blank value for the user to fill out.
+        fOptions = Lists.newArrayList("", "", "");
+    }
+    
+    @Override
+    public String getTypeName()
+    {
+        return "Multi-Select";
+    }
+    
+    /**
+     * Returns {@link Views#NEW_MULTI_SELECT_VARIABLE}.
+     */
+    @Override
+    public String getNewViewName()
+    {
+        return Views.NEW_MULTI_SELECT_VARIABLE;
+    }
+    
+    @Override
+    public ImmutableSortedSet<RetrievalEnum> getAllRetrievers()
+    {
+        return ImmutableSortedSet.of(RetrievalEnum.GENDER);
+    }
+    
+    /**
+     * Returns all possible display types for user selection.
+     * @return an immutable collection
+     */
+    public final ImmutableCollection<MultiSelectVariable.DisplayType> getAllDisplayTypes()
+    {
+        return ImmutableList.copyOf(MultiSelectVariable.DisplayType.values());
+    }
+    
+    /**
+     * Returns the DisplayType to set on the variable.
+     * @see MultiSelectVariable#getDisplayType()
+     */
+    public final MultiSelectVariable.DisplayType getDisplayType()
+    {
+        return fDisplayType;
+    }
+    
+    /**
+     * Sets the DisplayType to set on the variable.
+     * @see MultiSelectVariable#getDisplayType()
+     */
+    public final void setDisplayType(final MultiSelectVariable.DisplayType displayType)
+    {
+        fDisplayType = displayType;
+    }
+    
+    /**
+     * Returns the options to set on the variable. Note that {@link
+     * MultiSelectVariable}s actually have collections of {@link MultiSelectOption},
+     * but since those objects effectively only contain Strings, we represent
+     * them as Strings here for easy use in views.
+     * @see MultiSelectVariable#getOptions()
+     * @return a mutable List
+     */
+    public final List<String> getOptions()
+    {
+        return fOptions;
+    }
+    
+    /**
+     * Like {@link #getOptions()} but with trailing blanks trimmed off. (The
+     * HTTP form submission may create spurious trailing blank options.) The
+     * returned list may still contain empty strings in the middle.
+     * @return an ImmutableList
+     */
+    public final ImmutableList<String> getTrimmedOptions()
+    {
+        // Start with a copy of all the options.
+        final ArrayList<String> trimmed = new ArrayList<>(fOptions);
+        
+        // Iterate from the end, removing as we go.
+        for (int i = fOptions.size() - 1; i >= 0; --i)
+        {
+            if (Strings.isNullOrEmpty(trimmed.get(i)))
+            {
+                trimmed.remove(i);
+            }
+            // Break at the first non-empty String.
+            else
+            {
+                break;
+            }
```

Comment:
Shouldn't you iterate through all of the strings and remove the necessary ones, or is the comment stating that empty strings in the middle of the options are actually okay?